siddhi-testsuite/src/main/resources/archetype-resources/component/src/main/java/sample/containers/LoggerServiceContainer.java

```diff
@@ -0,0 +1,94 @@
+package sample.containers;
+
+import io.siddhi.distribution.test.framework.util.HTTPClient;
+import org.testcontainers.containers.ContainerLaunchException;
+import org.testcontainers.containers.GenericContainer;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.URI;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import static org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess;
+
+/**
+ * Siddhi Logger Service container class. This is a sample LoggerServiceContainer which can be used to log a message.
```

Comment:
Rephrase the comments.

Sample container extending the Container framework which logs events received through an HTTP endpoint.